### PR TITLE
feat(ISV-5128): add script to update component sboms

### PR DESCRIPTION
**Pull request by wcheang**

### CODE REVIEW

File: sbom/update_component_sbom
Comment (mmalina) [inline sbom/update_component_sbom:1]: Correct me if I'm wrong, but it looks like you created this as an actual file and not a symlink? At least that's what it looks like if I compare this to this PR: https://github.com/konflux-ci/release-service-utils/pull/254/files#diff-26bc81b0774ec9f78e7aa4ae5d90c3b8a4f13cac3a2c2a46b10d2217698830d1 - you can see a different icon for the file on the left.

File: sbom/update_component_sbom.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    Args:
+        data_path: path to data.json file containing image metadata.
+        input_path: path to directory holding SBOMs files to be updated.
+        output_path: path to directory to save updated SBOMs.
+    """
+    with open(data_path, "r") as data_file:
+        data = json.load(data_file)
+        release_notes = data["releaseNotes"]
+
+    component_to_purls_map = get_component_to_purls_map(release_notes["images"])
```
Comment (mmalina) [inline sbom/update_component_sbom.py:79]: Technically, only the `json.load` file needs the open file, right? And also, the rest of `release_notes` is never used, only images. So maybe you can simplify it like this?
```suggestion
    with open(data_path, "r") as data_file:
        data = json.load(data_file)

    component_to_purls_map = get_component_to_purls_map(data["release_notes"]["images"])
```

**Review by mmalina — COMMENTED** [review]
Only two small comments, but otherwise it looks good.

File: sbom/update_component_sbom
Comment (wcheang) [reply sbom/update_component_sbom:1]: You are correct. Somehow my brain saw what's done for other scripts and didn't read them as symlinks, and thought it was some repo-specific mechanism :sweat_smile: Fixed and added the proper symlink.

Comment (mmalina) [reply sbom/update_component_sbom:1]: ok, thanks, no worries :)

**Review by mmalina — APPROVED** [review]
You will need to rebase and then we can merge.

**Comment by wcheang** [comment]
Rebased and ran black on the test file.

**Review by mmalina — APPROVED** [review]

**Pull request merged**